src/messages/responses/set/cache-set.ts

```diff
@@ -0,0 +1,46 @@
+// older versions of node don't have the global util variables https://github.com/nodejs/node/issues/20365
+import {TextDecoder} from 'util';
+import {CacheSetResponse} from './cache-set-response';
+import {MomentoErrorCode, SdkError} from '../../../errors/errors';
+
+const TEXT_DECODER = new TextDecoder();
+
+export class Success extends CacheSetResponse {
+  private readonly value: Uint8Array;
+  constructor(value: Uint8Array) {
+    super();
+    this.value = value;
+  }
+
+  public valueString(): string {
+    return TEXT_DECODER.decode(this.value);
+  }
+
+  public valueBytes(): Uint8Array {
+    return this.value;
+  }
```

Comment:
Mixin to share with CacheGetResponse?

Reply:
See above regarding mixins . . . I'll look into that separately.